packages/compass-aggregations/src/modules/pipeline-builder/pipeline-parser/index.ts

```diff
@@ -1,3 +1,4 @@
 import PipelineParser from './pipeline-parser';
+import Stage from './stage';
 
-export { PipelineParser };
\ No newline at end of file
+export { PipelineParser, Stage };
```

Comment:
Do we need this re-export here? When I was trying this branch locally, this trips up ts autocomplete and in a few minutes I ended up with imports for this from two different places

Reply:
It didn't feel like Stage belongs in PipelineParser. So, moved it next to pipeline-builder. As a result, this cleaned up.